enclave/src/rpc/worker_api_direct.rs

```diff
@@ -24,55 +37,42 @@ use alloc::{
 	string::String,
 	vec::Vec,
 };
+use base58::FromBase58;
+use chain_relay::Block;
+use codec::{Decode, Encode};
 use core::{ops::Deref, result::Result};
-
+use jsonrpc_core::{futures::executor, Error as RpcError, *};
+use log::*;
 use sgx_types::*;
+use sp_core::H256 as Hash;
 use std::sync::{
 	atomic::{AtomicPtr, Ordering},
 	Arc, SgxMutex,
 };
-
-use sp_core::H256 as Hash;
-
-use codec::{Decode, Encode};
-use log::*;
-
-use crate::rpc::{
-	api::SideChainApi,
-	author::{Author, AuthorApi},
-	basic_pool::BasicPool,
-};
-
-use crate::top_pool::pool::Options as PoolOptions;
-
-use self::serde_json::*;
-use jsonrpc_core::{futures::executor, Error as RpcError, *};
-
-use substratee_stf::ShardIdentifier;
-
-use base58::FromBase58;
-use chain_relay::Block;
-
 use substratee_node_primitives::Request;
+use substratee_stf::ShardIdentifier;
 use substratee_worker_primitives::{
 	block::SignedBlock, DirectRequestStatus, RpcReturnValue, TrustedOperationStatus,
 };
 
-use crate::{ocall::rpc_ocall::EnclaveRpcOCall, rsa3072, utils::write_slice_and_whitespace_pad};
-
 static GLOBAL_TX_POOL: AtomicPtr<()> = AtomicPtr::new(0 as *mut ());
 
 #[no_mangle]
 // initialise tx pool and store within static atomic pointer
 pub unsafe extern "C" fn initialize_pool() -> sgx_status_t {
+	// doesn't do anything anymore, the top pool is initialized when the RPC direct server is initialized
```

Comment:
None apart from laziness and wanting to wrap up this PR asap 😄 Cleaning this up will take a bit of work, since it's an entire stack from e-call bridge, to EDL file, to this function here. I'll create an issue, so we don't forget.